README.md

```diff
@@ -136,6 +136,23 @@ Run migrations – this step will create tables for `auth` and `authtoken` apps:
 
     $ ./manage.py migrate
 
+### JSON Web Token Authentication
+`djoser` works out of the box with JSON web token authentication. You simply
+need to route correctly in your `settings.URLCONF`. An example would be:
+
+```
+import rest_framework_jwt.views
+import djoser.views
+
+urlpatterns = [
+    url(r'^auth/login', rest_framework_jwt.views.obtain_jwt_token),  # using JSON web token
+    url(r'^auth/register', djoser.views.RegistrationView.as_view()),
+    url(r'^auth/password/reset', djoser.views.PasswordRestView.as_view()),
```

Comment:
There is a typo in the view name, it should be:
`url(r'^auth/password/reset', djoser.views.PasswordResetView.as_view()),`